docs/tutorials/ad-hoc-simulation/planning-simulation.md

```diff
@@ -89,16 +89,26 @@ ros2 topic pub /autoware/engage autoware_auto_vehicle_msgs/msg/Engage "engage: t
 ### Placing dummy objects
 
 1. Click the `2D Dummy Car` or `2D Dummy Pedestrian` button in the toolbar.
-
 2. Set the pose of the dummy object by clicking and dragging on the map.
-
 3. Set the velocity of the object in `Tool Properties -> 2D Dummy Car/Pedestrian` panel.
 
-!!! note
+   !!! note
```

Comment:
This note is not rendered properly :disappointed: 
Can you try to use a 2 spaces indent ? (https://github.com/squidfunk/mkdocs-material/discussions/3763)

Reply:
I did this but then pri-commit.ci failed

Reply:
I added commit 38f8f9f11 to prevent `markdownlint` and `prettier` from breaking the indentation.

@kenji-miyake I could not find a better solution. I tried to use `<!-- prettier-ignore -->` (https://prettier.io/docs/en/ignore.html#markdown) but it did not work.

Reply:
I think it works with 4 spaces.

Reply:
When building the doc locally 4 spaces produce the following:
![image](https://user-images.githubusercontent.com/78338830/201024022-072ea3e4-1521-49e0-b9a7-7adbaa428733.png)

No indentation works but breaks the list numbering.

As far as I could find, 2 spaces is the only indentation that produces a correct `note` and does not break numbering.

Reply:
Okay, we need to disable `mdx_truly_sane_lists` to display admonition correctly. I'll check the reason why we added it.

https://github.com/squidfunk/mkdocs-material/discussions/3763#discussioncomment-2833731

Reply:
I'll try improving `markdownlint`, but if you are in hurry, you can revert my change and merge this PR with Maxime-san's workaround.

Reply:
I've looked into this again, but it was a really complex problem. :pleading_face: 

First, the following style works.
![image](https://user-images.githubusercontent.com/31987104/209193440-32930fb8-3535-4ea1-aad0-e98da0836393.png)

But there are several issues.
- markdownlint raises warnings.
- Prettier doesn't allow this format and re-formats to the following style (removes spaces), which doesn't work because the number of the ordered list is reset.
![image](https://user-images.githubusercontent.com/31987104/209193914-fc602058-3f89-4534-ada9-35a8e5da1c39.png)

Reply:
Also, removing `mdx_truly_sane_lists` or changing the indent size to 4 might work, but will cause another problem. :pleading_face: 

So in conclusion, my proposal is to stop using admonition inside ordered lists and wait until the tool's compatibility problems are fixed.